openscap-policies/src/main/scala/com/normation/plugins/openscappolicies/api/OpenScapApi.scala

```diff
@@ -0,0 +1,146 @@
+package com.normation.plugins.openscappolicies.api
+
+import com.normation.box._
+import com.normation.inventory.domain.NodeId
+import com.normation.plugins.openscappolicies.OpenscapPoliciesLogger
+import com.normation.plugins.openscappolicies.services.OpenScapReportReader
+import com.normation.plugins.openscappolicies.services.ReportSanitizer
+import com.normation.rudder.api.HttpAction.GET
+import com.normation.rudder.rest.EndpointSchema.syntax._
+import com.normation.rudder.rest._
+import com.normation.rudder.rest.lift.DefaultParams
+import com.normation.rudder.rest.lift.LiftApiModule
+import com.normation.rudder.rest.lift.LiftApiModuleProvider
+import com.normation.rudder.rest.ApiModuleProvider
+import com.normation.rudder.rest.EndpointSchema
+import com.normation.rudder.rest.GeneralApi
+import com.normation.rudder.rest.SortIndex
+import com.normation.rudder.rest.StartsAtVersion10
+import net.liftweb.common.Box
+import net.liftweb.common.EmptyBox
+import net.liftweb.common.Full
+import net.liftweb.http.InMemoryResponse
+import net.liftweb.http.LiftResponse
+import net.liftweb.http.Req
+import net.liftweb.json.JValue
+import net.liftweb.json.NoTypeHints
+import sourcecode.Line
+
+sealed trait OpenScapApi extends EndpointSchema with GeneralApi with SortIndex
+object OpenScapApi extends ApiModuleProvider[OpenScapApi] {
+
+  final case object GetOpenScapReport extends OpenScapApi with OneParam with StartsAtVersion10 {
+    val z = implicitly[Line].value
+    val description    = "Get OpenScap report for a node"
+    val (action, path) = GET / "openscap" / "report" / "{id}"
+  }
+
+  final case object GetSanitizedOpenScapReport extends OpenScapApi with OneParam with StartsAtVersion15 {
```

Comment:
it should be version 10

Reply:
actually, it must be 15 :)

Reply:
why? It is not in supported API (we stop at 12 in rudder)

Reply:
15 is supported, and all techniques resources and parameters endpoint are already at level 15